main/test/wallet-backend.test.js

```diff
@@ -47,8 +47,19 @@ describe('Wallet Backend', function () {
     it('fetches all transactions', /** @this {Mocha.Test} */ async function () {
       this.timeout(20_000)
 
+      await backend.setup(
+        // eslint-disable-next-line max-len
+        'insane believe defy best among myself mistake account paddle episode life music fame impact below define habit rotate clay innocent history depart slice series'
```

Comment:
It's probably obvious, but I want to call it out explicitly:

All wallets derived from this seed are public, meaning that anyone can sign transactions using wallet's private key.